build_scripts/build.py

```diff
@@ -881,14 +910,19 @@ def execbuildscript(filepath):
 		"-DDEPENDENCY_LUA_LIBRARY=" +lua_jit_lib +""
 	]
 
+cmake_args.append("-DPME_EXTERNAL_LIB_LOCATION=" +external_libs_dir)
```

Comment:
@SlawekNowy What's PME?

Reply:
You've defined these yourself in the modules. This is added here early. I merely copied this name from `pr_source`